UI/assets/css/style.css

```diff
@@ -453,13 +455,23 @@ header {
 
 #transaction-history-overlay-content div p {
   width: 100%;
-  height: 30px;
+  line-height: 25px;
   color: #013144;
   font-size: 16px;
   margin-top: 0;
+  padding-left: 20px;
   margin-bottom: 10px;
 }
 
+.modal-full-div {
+  width: 100%;
+}
+
+.modal-half-div {
+  width: 50%;
+  float: left;
+}
+/*** *** *** *** *** ***/
```

Comment:
what is this line here for?

Reply:
That's what I used to separate the text in the transaction details modal into two parts. I put each entry into a "modal-full-div" which is like the row, and seperated them by putting the key e.g "ID" and value e.g "12345" into separate divs "modal-half-div" and floated the half div so they can be separate from each other.

Reply:
If it's the comment line you're referring to, it was just used to separate the styling above from the section below it. Maybe I shouldn't have spaced the asterisks.